src/metatensor/models/utils/export.py

```diff
@@ -13,10 +13,17 @@ def export(model: torch.nn.Module, output: str) -> None:
     :param output: Path to save the exported model
     """
 
+    if model.capabilities.length_unit == "":
```

Comment:
I have no opinion on this. Both are not super insightful.... @frostedoyster what do you think?

Reply:
if @frostedoyster is ok with this for me the pr is ok

Reply:
I think `""` is more useful to the reader because it makes it clear that the variable is a string. Using len() that could be anything (list, torch.Tensor, dict,...)